hr_employee_ppe/__manifest__.py

```diff
@@ -3,23 +3,21 @@
 
 {
     "name": "Personal Protective Equipment (PPE) Management",
-    "version": "13.0.1.0.0",
+    "version": "13.0.2.0.0",
     "author": "Escodoo, Odoo Community Association (OCA)",
     "maintainers": ["marcelsavegnago", "eduaparicio"],
     "images": ["static/description/banner.png"],
     "website": "https://github.com/OCA/hr",
     "license": "AGPL-3",
     "category": "Human Resources",
-    "depends": ["hr", "mail", "product"],
+    "depends": ["hr_personal_equipment_request"],
```

Comment:
I don't think this change of dependencies in a stable version is correct. And more to a non standard module like `hr_personal_equipment_request`.

Reply:
Why non standard?? hr_personal_equipment_request allows to manage all kind of equipment requests, like job dress, shoes and so on, not only ppes. I thin it is more generic and it was defined with the original creator of the module. I understand that it might be necessary to apply the change on a version change, not in a implemented version. But it has sense to apply the change IMO

Reply:
I mean that is not an Odoo SA module. I don't know about the features, but it sound a very heavy change for this module. Why requests should be a subset of this module?

Reply:
Aaah! I understand your concern. The thing is, ppe is a special case of material delivered to your employees, so we splitted tge functionality in two, also we made some glue modules wit tier validation and stock. The main reason for the splitting comes from the legal requirements attached to ppes (formation, delivery and so on). This legal requirmlements are not needed on non ppe(like a shit with your company logo)

Also, on the change some design desitions were remade in order to simplify models

Reply:
If the users of this module agree to do this change, I won't oppose then.

Reply:
For my part, we can continue with the change in dependencies.